.jenkins/pytorch/test.sh

```diff
@@ -42,6 +42,15 @@ if [[ "$BUILD_ENVIRONMENT" != *ppc64le* ]]; then
   # TODO: move this to Docker
   pip install -q hypothesis --user
 
+  # TODO: move this to Docker
+  python_version=`python -c 'import platform; print(platform.python_version())'|cut -c1`
+  echo $python_version
+  if [[ $python_version == "2" ]]; then
+    pip install -q http://m100.nthu.edu.tw/~s100062805/tensorboard/tensorboard-1.14.0a0-py2-none-any.whl --user
+  else
+    pip install -q http://m100.nthu.edu.tw/~s100062805/tensorboard/tensorboard-1.14.0a0-py3-none-any.whl --user
```

Comment:
I've reuploaded these wheels to https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-py2-none-any.whl and https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-py3-none-any.whl please use these URLs instead.